vars/daosPackagesVersion.groovy

```diff
@@ -18,7 +18,7 @@ String daos_latest_version(String next_version, String repo_type='stable') {
     String v = sh(label: "Get RPM packages version",
                   script: 'repoquery --repofrompath=daos,' + env.REPOSITORY_URL +
                           env["DAOS_STACK_EL_7_" + repo_type.toUpperCase() + "_REPO"] +
-                        '''/ --repoid daos --qf %{version}-%{release} --whatprovides 'daos(x86-64) < ''' +
+                        '''/ --repoid daos --qf %{version}-%{release} --whatprovides 'daos-tests(x86-64) < ''' +
```

Comment:
Just curious why this needed changing.  My only concern is that the `daos` package is probably a more stable name than `daos-tests`.  The latter for example could be ripe to be completely replaced by a bunch of `daos-tests-$subpackages` and then this function will fail due to a missing `daos-tests` package.